utils/openwrt-exporter/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2013-2017 OpenWrt.org
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openwrt-exporter
+PKG_VERSION:=0.1.0
```

Comment:
If PKG_SOURCE_VERSION does not refer to a tagged release in the source repo, we often use a scheme like *2017.05.07. PKG_SOURCE_VERSION* as the version number. Not a big deal, just noticed it. This is to avoid confusion when looking at upstream code.

I made the same mistake a while ago. Check it out to learn from it and make it better than I did. See the comment from @yousong who explained it in more detail: https://github.com/openwrt/packages/pull/3666#pullrequestreview-13443316